tests/features/project.feature

```diff
@@ -1,4 +1,4 @@
-@api
+@api @project
```

Comment:
Is this tag has any related function? I can't see any.

Reply:
none, removed